src/mrpro/operators/models/_MOLLI.py

```diff
@@ -16,27 +16,35 @@
 
 
 class MOLLI(Operator):
+    """Signal model for Modified Look-Locker inversion recovery (MOLLI)."""
+
     def __init__(self, ti: torch.Tensor):
-        """Parameters needed for MOLLI.
+        """Initialize MOLLI signal model for T1 mapping.
 
         Parameters
         ----------
         ti
-            inversion times
+            inversion times [s]
         """
         super().__init__()
         self.ti = torch.nn.Parameter(ti, requires_grad=ti.requires_grad)
 
     def forward(self, a: torch.Tensor, b: torch.Tensor, t1: torch.Tensor) -> torch.Tensor:
-        """Apply the forward model.
+        """Forward method of the MOLLI signal model.
 
         Parameters
         ----------
-            Quantitative parameters m0, t1 with dimensions (other, c, z, y, x)
+        a
+            Parameter a in MOLLI signal model
+        b
+            Parameter b in MOLLI signal model
+        t1
+            longitudinal relaxation time T1 [s]
 
         Returns
         -------
-            Image data tensor (other, c, z, y, x)
+        torch.Tensor
+            data tensor with dimensions ((... inv_times), coils, z, y, x)
```

Comment:
and similiar...

Reply:
I didn't change this yet.
Will check the numpy docstring style documentation again.

Reply:
We should try to aim to keep this consistent in our code base. 

I am pretty sure currently more function do not specify the return type in the docstring.

If you decide one way or the other, it would be great to change it everywhere.

Reply:
removed the return types ✔️